conans/model/conf.py

```diff
@@ -82,6 +82,7 @@
     "tools.files.download:retry": "Number of retries in case of failure when downloading",
     "tools.files.download:retry_wait": "Seconds to wait between download attempts",
     "tools.files.download:verify": "If set, overrides recipes on whether to perform SSL verification for their downloaded files. Only recommended to be set while testing",
+    "tools.graph:bundle": "(Experimental, boolean, default False) Enable the computation of dependencies of package bundles",
```

Comment:
I was thinking that maybe `tools.graph:unbundle` would be more appropriate?

Reply:
Alternatives I have considered too:
- ``expand_bundle``
- ``create_bundle``
- ``build_bundle``

I think ``unbundle`` can be a bit more confusing, but ``tools.graph:build_bundle=True`` is more explicit and clear (it comes after an error if you try to build it from source)

Reply:
Renamed to ``build_bundle``